amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -428,13 +434,71 @@ def test_getAGKitIDsByEmail(self):
         raise NotImplementedError()
 
     def test_ag_set_pass_change_code(self):
-        raise NotImplementedError()
+        # Generate new random code and assign it
+        testcode = ''.join(choice(ascii_letters) for i in range(10))
+        self.ag_data.ag_set_pass_change_code('REMOVED', 'tst_ULGcr', testcode)
+
+        # Acutally test the change
+        obs = self.ag_data.ag_verify_kit_password_change_code(
+            'REMOVED', 'tst_ULGcr', 'SOMELONGTHINGTHATWILLFAIL')
+        self.assertEqual(obs, False)
+        obs = self.ag_data.ag_verify_kit_password_change_code(
+            'REMOVED', 'tst_ULGcr', testcode)
+        self.assertEqual(obs, True)
+
+        # Test giving nonsense email
+        # TODO: make this raise error and test
+        self.ag_data.ag_set_pass_change_code('Fake@notarealemail.com',
+                                             'tst_ULGcr', testcode)
+
+        # Test giving bad skid
+        # TODO: make this raise error and test
+        self.ag_data.ag_set_pass_change_code('REMOVED', 'NOTINTHEDB', testcode)
```

Comment:
That is all done above. These two are in to make sure that when the error raising is added, the test fails and we know to edit the test.